### ISSUE TITLE
No "un-install" doc for logging tools

### ISSUE THREAD

**Issue by ckden** [newrelic/docs-website#4118]
### Tell us what you need

Hi, first great job on explaining and documenting the various ways to bring data into New Relic logging.  However, once these things are setup, the process to unwind them does not seem to be easy to identify.    In particular, if I have a application team who is sending log test data to the logs.API (For example), it would be helpful to have something documented that app teams could follow to understand how to unconfigure a log ingestion cleanly.  

we have several apps teams who need help with this and I am unable to provide a simple answer since each is using different methods to ingest log info. 

Thank you for your consideration of this request. 

### Anything else you'd like to share?

Most of your other products have an uninstall or removal feature defined through documentation, adding this would be more consistent and help teams avoid ingestion of log information that is no longer needed. 

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent/
* **OS**: Windows 10
* **Browser**: Chrome 93.0.4577.82
* **Device**: Unknown

**Comment by bradleycamacho** [comment]
Hey @ckden, thanks for the very thoughtful feedback! This is a great call, and we can explore documenting this with our product teams. I am going to tag in @barbnewrelic, one of our docs writers with more context about logs.

**Comment by barbnewrelic** [comment]
@ckden  I reviewed our backlog of log doc requests and discovered this was on the roadmap 2 years ago, but then the team decided not to document uninstall procedures at that time. Thanks for raising this issue again. I'll get it back into the queue!

**Comment by barbnewrelic** [comment]
@ckden  Do you only need procedures to uninstall/stop Log API, or are there other log forwarders your teams currently use? There are several different ways to get logs into New Relic, and the logging SMEs want to prioritize the ones you use first. Thanks for confirming!

**Comment by barbnewrelic** [comment]
@ckden  We are happy you are using so many options for logs, and we are also sad that uninstall procedures never got documented. I'm going to file a ticket for the documentation work, but I don't think I can file a support ticket on your behalf to get immediate help from support. I've already notified them about your question, but I wonder if it would help expedite if the request for immediate assistance was coming directly from you as a support ticket? I do apologize to give you extra work, but I don't want to keep you waiting on our docs team. The uninstall procedures will have to come from our tech support engineers, and we will then be able to use their solutions to publish in the documentation site. I hope this gives you the help you need for the short term!

**Comment by ckden** [comment]
Hi again @barbnewrelic , understood and no problem.  I will open one this evening as a follow up to this thread.   Thank you.

**Comment by barbnewrelic** [comment]
Added info to all the other log forwarder docs with https://github.com/newrelic/docs-website/pull/6280
